src/components/NetworkModal/index.tsx

```diff
@@ -85,6 +85,8 @@ export default function NetworkModal(): JSX.Element | null {
 
   if (!chainId || !networkModalOpen) return null
 
+  const listNetwork = process.env.NODE_ENV === 'production' ? MAINNET_NETWORKS : SUPPORTED_NETWORKS
```

Comment:
move this out of the component

Reply:
maybe rename to `networks`?

Reply:
If move out of the component, please dont name it `networks`. It's very bad naming and doesnt provide much infomation since there are many lists of networks for different purpose. At least name it `SHOW_NETWORKS` or similar.